src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptStreamingFindUsagesPresenter.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.Editor.Host;
+using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    [Export(typeof(IVSTypeScriptStreamingFindUsagesPresenter)), Shared]
+    internal sealed class VSTypeScriptStreamingFindUsagesPresenter : IVSTypeScriptStreamingFindUsagesPresenter
+    {
+        private readonly IStreamingFindUsagesPresenter _underlyingObject;
+
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public VSTypeScriptStreamingFindUsagesPresenter(IStreamingFindUsagesPresenter underlyingObject)
+            => _underlyingObject = underlyingObject;
+
+        public (VSTypeScriptFindUsagesContext context, CancellationToken cancellationToken) StartSearch(
+            string title, bool supportsReferences)
+        {
+            var (context, cancellationToken) = _underlyingObject.StartSearch(title, supportsReferences);
+            return (new(context), cancellationToken);
+        }
+
+        public (VSTypeScriptFindUsagesContext context, CancellationToken cancellationToken) StartSearchWithCustomColumns(
```

Comment:
❔ Are both of these required for TS right now? If not, we should remove the ones that are not yet needed.

Reply:
Just the first one. Will remove.

Reply:
It's only a problem to add to an interface if TypeScript is implementing the interface. Since the implementation here is in Roslyn, there would be no problem adding to it in the future if necessary.

Reply:
You're right. Didn't realize that. I'll remove the method.